source/ShopPanel.cpp

```diff
@@ -1220,14 +1239,56 @@ void ShopPanel::SideSelect(Ship *ship)
 		}
 	}
 	else if(!control)
+	{
 		playerShips.clear();
-	else if(playerShips.contains(ship))
+		if(clicks > 1)
+			for(const shared_ptr<Ship> &it : player.Ships())
+			{
+				Ship *itShip = it.get();
+				if(itShip != ship && itShip->Immitates(*ship))
+					playerShips.insert(itShip);
```

Comment:
Same changes as below:
```suggestion
				if(!CanShowInSidebar(*it, player.GetPlanet()))
					continue;
				if(it.get() != ship && it->Immitates(*ship))
					playerShips.insert(it.get());
```